src/dmd/dversion.d

```diff
@@ -199,3 +199,17 @@ extern (C++) final class VersionSymbol : Dsymbol
         v.visit(this);
     }
 }
+
+private Module moduleOf(ScopeDsymbol sds)
+{
+    Dsymbol s = sds;
+    while (s.isNspace())
+    {
+        if (Module m = s.isModule())
+            return m;
+        s = s.parent;
+    }
+    if (Module m = s.isModule())
+        return m;
+    return null;
+}
```

Comment:
The implementation looks quite specific but the API looks quite generic.

Reply:
I'm not sure what you mean by this, but I changed the function name to better reflect what it does.